src/composables/draw/drawn-features.composable.ts

```diff
@@ -0,0 +1,29 @@
+import useMap from '../map/map.composable'
+import { Collection, Feature } from 'ol'
+import { Geometry } from 'ol/geom'
+import VectorLayer from 'ol/layer/Vector'
+import VectorSource from 'ol/source/Vector'
+
+export default function useDrawnFeatures() {
+  const map = useMap().getOlMap()
+
+  const features = new Collection<Feature<Geometry>>()
+  const drawLayer = new VectorLayer({
+    source: new VectorSource({
+      features,
+    }),
+    zIndex: 1000,
```

Comment:
Maybe we should put this one into a const like
` const DEFAULT_BGZINDEX = -200` ?